.github/workflows/bin-solr-test.yml

```diff
@@ -9,6 +9,7 @@ on:
       - '.github/workflows/bin-solr-test.yml'
       - 'solr/bin/**'
       - 'solr/packaging/**'
+      - 'solr/core/src/java/org/apache/solr/cli/**'
```

Comment:
@epugh I added the change I proposed in https://github.com/apache/solr/pull/2573#pullrequestreview-2188208038 to trigger bats tests for all changes in cli code.

NOTE this is not in main, so perhaps add it in another PR?

Reply:
Never mind, reverted.